plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFunctionFunctions.java

```diff
@@ -40,21 +41,62 @@ private FissionFunctionFunctions()
 
     @ScalarFunction("fission_dnsdb")
     @Description("Converts the string to alternating case")
-    @SqlType(StandardTypes.VARCHAR)
-    public static Slice dnsdb(@SqlType(StandardTypes.VARCHAR) Slice slice)
+    @SqlType(StandardTypes.BIGINT)
+    public static long dnsdb(@SqlType(StandardTypes.VARCHAR) Slice slice) throws JSONException, IOException
```

Comment:
I'm tempted to say let's rename the method to "DnsDb" unless we're deliberately lower-casing it?

Reply:
I agree we should rename this, but the name also needs to indicate some sort of action.
'fetchDnsDb' or 'dnsDbLookup' would be decent candidates.